packages/dart/lib/src/network/parse_query.dart

```diff
@@ -435,20 +435,6 @@ class QueryBuilder<T extends ParseObject> {
     return queryBuilder;
   }
 
-  String concatenateArray(List<String> queries) {
```

Comment:
Why is this removed?

Reply:
transferred it to parse_utils.dart file for access in other files

Reply:
Got it, I've overlooked that